src/store/index.ts

```diff
@@ -133,12 +133,12 @@ export const store = createStore({
          *  --> all recentSearches are store in localStorage and allow client side caching.
          */
         async SEARCH_IDENTITIES(context: ActionContext<StoreI, StoreI>, { searchData, currentPage }): Promise<void> {
+            context.commit("incrementBusyCounter");
             const wsAddress = getChainAddress(searchData.selectedChainKey);
             if (!wsAddress) {
                 throw new Error("[store/index] No address given for chain: " + searchData.selectedChainKey);
                 return console.error("[store/index] No address given for chain: ", searchData.selectedChainKey);
```

Comment:
this return is not reachable. Please remove.

Reply:
I didn't add this line. But I already removed it in other branch I requested your review :)